src/components/layouts/FilterBar.tsx

```diff
@@ -18,7 +31,60 @@ export const FilterBar = () => {
     >
       <AppMargin size={SizeofAppMargin.LARGE} py="18.5px">
         <Stack direction="row" alignItems="center" justifyContent="space-between">
-          <div> </div>
+          <Stack direction="row" columnGap={3}>
+            <Selector
+              getSelectedValue={(_newValue) => {
+                const newValue = _newValue as IAssigneeCombined
+                updateAssigneeValue(newValue)
+                store.dispatch(setFilteredAsignee(_newValue))
+              }}
+              startIcon={<FilterByAsigneeIcon />}
+              options={assignee}
+              placeholder="Assignee"
+              value={assigneeValue}
+              selectorType={SelectorType.ASSIGNEE_SELECTOR}
+              customOptions={{ id: '', name: 'No assignee', value: '' }}
+              buttonContent={
+                <Typography
+                  variant="bodySm"
+                  lineHeight="32px"
+                  fontWeight={500}
+                  fontSize="12px"
+                  sx={{ color: (theme) => theme.color.gray[600] }}
+                >
+                  <Stack direction="row" alignItems="center" columnGap={1}>
+                    <> Filter by</>
+                    {assigneeValue?.name || assigneeValue?.givenName ? (
+                      <Stack direction="row" alignItems="center" columnGap={1}>
+                        <Avatar
+                          alt="user"
+                          src={
+                            (assigneeValue as IAssigneeCombined).avatarImageUrl ||
+                            (assigneeValue as IAssigneeCombined).iconImageUrl
+                          }
+                          sx={{ width: '20px', height: '20px' }}
+                        />
+                        {assigneeValue?.name || assigneeValue?.givenName}
+                        <IconButton
+                          aria-label="remove"
+                          onClick={(e) => {
+                            e.stopPropagation()
+                            updateAssigneeValue(null)
+                            store.dispatch(setFilteredAsignee(null))
+                          }}
+                        >
+                          <CrossIcon />
+                        </IconButton>
+                      </Stack>
+                    ) : (
+                      <> assignee</>
+                    )}
+                  </Stack>
+                </Typography>
+              }
```

Comment:
Create a separate component as `FilterButton` or similar to place this part of the code. This component will be responsible for the behaviour rendered by this code.